Emby.Server.Implementations/Networking/NetworkManager.cs

```diff
@@ -414,7 +414,7 @@ private List<IPAddress> GetIPsDefault()
                     // Try to exclude virtual adapters
                     // http://stackoverflow.com/questions/8089685/c-sharp-finding-my-machines-local-ip-address-and-not-the-vms
                     var addr = ipProperties.GatewayAddresses.FirstOrDefault();
-                    if (addr == null || string.Equals(addr.Address.ToString(), "0.0.0.0", StringComparison.OrdinalIgnoreCase))
+                    if (addr == null || (ignoreVirtualInterface && string.Equals(addr.Address.ToString(), "0.0.0.0", StringComparison.OrdinalIgnoreCase)))
```

Comment:
Redundant parenthesis

Reply:
I think it's fine, but you point out, I will update it soon.